src/run-browser-service.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2020-present, Elastic NV
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+import { BrowserService } from './core/browser-service';
+
+const browserService = new BrowserService();
+browserService.init();
```

Comment:
this should be moved inside 
```
 if (require.main === module) {
    const browserService = new BrowserService();
    browserService.init();
  }
```

to make sure we are running this file only when invoked via node src/run-browser-service

Reply:
I don't expect that we would require `run-browser-service.ts` if we do it's a bug, it's not meant to be imported. I see this similar to how the `cli.ts` works, it's expected to executed directly and not imported.

Reply:
Its a good practise to wrap things inside this pattern to avoid the consequence of auto initialising browser service. CLI is different. However this is a not a must do.

Reply:
Let's keep it as is then, I don't anticipate an issue here and it's an easy fix to do later on. There are plenty of libraries like us without this. FWIW I don't think this is a bad practice though, just unnecessary IMO.